lib/commands/element.js

```diff
@@ -15,9 +15,13 @@ commands.getAttribute = async function (attribute, el) {
       return await this.executeAtom('get_attribute_value', [atomsElement, attribute]);
     }
   } else {
-    if (_.includes(['label', 'name', 'value', 'values', 'hint'], attribute)) {
-      let command = `au.getElement('${el}').${attribute}()`;
-      return await this.uiAutoClient.sendCommand(command);
+    if (_.includes(['label', 'name', 'value', 'values', 'hint', 'contentSize'], attribute)) {
+      if (attribute === 'contentSize') {
```

Comment:
This could be switched around to make more sense, I think:
```js
if (attribute === 'contentSize') {
  return await this.getElementContentSize(el);
}
if (_.includes(['label', 'name', 'value', 'values', 'hint'], attribute)) {
  let command = `au.getElement('${el}').${attribute}()`;
  return await this.uiAutoClient.sendCommand(command);
}
```